frontend/src/components/table/Table.tsx

```diff
@@ -0,0 +1,34 @@
+import { Table as ATable, TableProps } from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import { size } from '../../constants';
+
+import { Colors } from '../../styles';
+
+const StyledTable = styled(ATable)`
+    box-shadow: 0px 5px 30px -6px rgba(100, 100, 100, 0.5);
```

Comment:
Sam kolor idealnie byłoby wydzielić do zmiennej `colors` jako nową stałą, jego nazwa to np. `shadow` (tylko tutaj trzeba zamienić rgba na rgb i utworzyć nowy kolor)